### PR TITLE
Secure Boot related refine

### PR DESCRIPTION
**Pull request by BaochengSu**
- Re-organize the secure boot keys
- Add OTP provisioning to menu
- Warn usage of dummy key

### CODE REVIEW

**Review by jan-kiszka — COMMENTED** [review]
file://dir is fine with links inside dir. file://dir/* had problems, no? I don't see the need for explicit key listing yet.

File: Kconfig [hunk 1]
```diff
@@ -181,3 +203,53 @@ config FIRMWARE_SECURE_VER
 	help
 	  Use specific anti-rollback secure version rather than the default 0.
 	  Range 0 - 127.
+
+choice
+	prompt "*** OTP provisioning command type ***"
```
Comment (jan-kiszka) [inline Kconfig:208]: This `***` separator does not look very beautiful IMHO.

Comment (BaochengSu) [reply Kconfig:208]: image

Comment (BaochengSu) [reply Kconfig:208]: image

Comment (jan-kiszka) [reply Kconfig:208]: Ah, was me in fact :smile:

**Comment by jan-kiszka** [comment]
You need to describe better the boundary conditions when this starts to break. Currently, it works. I suppose it fails when the links we have in keys/ are pointing outside of the keys/ folder, right?

**Comment by BaochengSu** [comment]
> You need to describe better the boundary conditions when this starts to break. Currently, it works. I suppose it fails when the links we have in keys/ are pointing outside of the keys/ folder, right?

Yep.

**Review by jan-kiszka — APPROVED** [review]

**Pull request merged**